sdk/monitor/Azure.Monitor.Query/README.md

```diff
@@ -187,14 +187,19 @@ foreach (var row in table.Rows)
 
 ### Increase query timeout
 
-Some queries take longer to execute than the default service timeout allows. You can use the `LogsQueryOptions` parameter to specify the service timeout.
+Some Logs queries take longer than 3 minutes to execute. The default server timeout is 3 minutes. You can increase the server timeout to a maximum of 10 minutes. In the following example, the `LogsQueryOptions` object's `ServerTimeout` property is used to set the server timeout to 5 minutes:
 
 ```C# Snippet:QueryLogsPrintTable
```

Comment:
I think this is pointing to the wrong snippet. We have a Snippet:QueryLogsWithTimeout:

https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.Query/tests/LogsClientSamples.cs#L171

Code blocks that have `Snippet:` in them are inserted from source files. You can update md filed by running ` .\eng\scripts\Update-Snippets.ps1 monitor` the CI automation would check that the cs and md files are in sync.

Reply:
Thanks for the explanation. Updated.